### ISSUE TITLE
Constructed Stats not showing Casual matches

### ISSUE THREAD

**Issue by Gr3yz** [HearthSim/Hearthstone-Deck-Tracker#2558]
When I go to Stats -> Constructed -> Matches, Casual matches do not show in the list, only ranked ones do.

I have my settings set to record Casual.  The win/less record for the decks still updates in the main window from casual games, but they just don't show in the stats page.  

Any advice on how to fix this please?  I think it stopped working 2-3 patches ago.    

I have attached screen shots showing my Stats page (showing just ranked games) and my settings:

![untitled](https://cloud.githubusercontent.com/assets/17360564/15694889/b9f676fa-276f-11e6-88b9-8d994037821c.png)
![untitled1](https://cloud.githubusercontent.com/assets/17360564/15694890/b9f740bc-276f-11e6-861b-d4410b061dd7.png)

**Comment by Gr3yz** [comment]
I figured it out.  FYI:

In the "Stats" page, if you change the "Rank" filter (I had changed it from L1 to 5), and afterwards change the game "Mode: filter back to "All" or "Casual", it seems to still use the set "Rank" filter, and not show any casual games as a result.  To fix it, I had to go change the Mode back to Ranked, then change the Rank filter back to L1 to 25, then change the Mode filter back to All/Casual for my Casual games to show again.
